requirements.txt

```diff
@@ -30,7 +30,7 @@ pytest
 fs==2.3.0
 python-slugify
 tftpy
-freezegun
+freezegun!=0.3.13
```

Comment:
@srenfo Thanks for the contribution! :slightly_smiling_face: Maybe it would be better to pin the exact  version? Maybe `0.13.2`?

Reply:
As far as I know 0.3.13 is the only broken version. It may be worth a comment in the file.

Version pinning is certainly useful, but it also creates a bunch of additional maintenance work. I don't know if anyone would be willing to sign up for that. Maybe this should be a separate discussion/issue?

Reply:
Can you add a comment in that case? :slightly_smiling_face: For future reference.